dbms/src/AggregateFunctions/AggregateFunctionNull.h

```diff
@@ -429,12 +429,19 @@ class AggregateFunctionNullUnary final : public AggregateFunctionNullBase<result
         if constexpr (input_is_nullable)
         {
             const ColumnNullable * column = static_cast<const ColumnNullable *>(columns[0]);
+
             if (!column->isNullAt(row_num))
             {
                 this->setFlag(place);
                 const IColumn * nested_column = &column->getNestedColumn();
                 this->nested_function->add(this->nestedPlace(place), &nested_column, row_num, arena);
             }
+            else if (const String & nested_func_name = this->nested_function->getName(); nested_func_name == "groupBitOr" || nested_func_name == "groupBitAnd" || nested_func_name == "groupBitXor")
```

Comment:
Can we use `if constexpr (result_is_nullable)`?

Reply:
I don't know if it is right for non-bitwise functions.